CRM/Contact/Form/Task/PDFLetterCommon.php

```diff
@@ -225,6 +225,7 @@ public static function buildQuickForm(&$form) {
   public static function setDefaultValues() {
     $defaultFormat = CRM_Core_BAO_PdfFormat::getDefaultValues();
     $defaultFormat['format_id'] = $defaultFormat['id'];
+    $defaultFormat['from_email_address'] = key(CRM_Core_OptionGroup::values('from_email_address', TRUE, FALSE, FALSE, ' AND is_default = 1'));
```

Comment:
How about 

```
$defaultFormat['from_email_address'] = civicrm_api3('OptionValue', 'getvalue', array(
  'return' => "label",
  'option_group_id' => "from_email_address",
  'is_default' => 1,
));
```

Reply:
@pradpnayak Agree that would be better to use the API there as it allows more flexibility in changes in the future. @yashodha Are you happy to make that change?